axelrod/strategies/titfortat.py

```diff
@@ -386,3 +386,33 @@ def reset(self):
         Player.reset(self)
         self.contrite = False
         self._recorded_history = []
+
+
+class SlowTitForTwoTats(Player):
+    """
+    A player that plays [c,c], then if the opponent plays the same move twice, 
```

Comment:
Could we write "A player that plays C twice, ...". This is not terribly important: just looks nicer to me :)